src/main/java/frc/trigon/robot/subsystems/turret/TurretConstants.java

```diff
@@ -0,0 +1,6 @@
+package frc.trigon.robot.subsystems.turret;
+
+public class TurretConstants {
+    static final double GEAR_RATIO = 100;
+    static final double MOMENT_OF_INERTIA = 0.003;
```

Comment:
Why is this here?

Reply:
it's only there for now, I'll move it when I use it

Reply:
...